### DISCUSSION TITLE
Fehler 46  /  Hoymiles HMT 2250

### DISCUSSION THREAD

**Discussion by Alf-Ele** [tbnobody/OpenDTU#1266] [Q&A]
Hallo zusammen,
ich betreibe (Ost-West) 2 Hoymiles HMT 2250 mit je 6 Solar Modulen a 370W.
Bei wechselnden Verhältnissen von direkter Sonneneinstrahlung und weißen Wolken (ideal für maximale Leistung) bekomme ich den Fehler 46. Heute bei 2130W auf West gegen 14Uhr.
Immer nur am Wechselrichter der Westseite.
Firmware openDTU ist aktuell (war bis eben 23.8.8) jetzt 23.8.22
Trotzdem funktioniert die Anlage.
Hat jemand eine Info für mich was Fehler 46 konkret bedeutet?
Danke und Grüße
alf.ele

**Comment by mattespiratos** [comment]
Hi,

ist bei mir mit 4 Modulen am gleichen WR auch heute passiert das ein MPPT dann quasi keine Leistung mehr liefert und er nur auf 2en weiterläuft. 

Ohne aber an die Leistungsgrenze zu kommen.

VG

**Comment by Solarteur** [comment]
Wenn der "Fehler 46" einem bestimmten Ereignis zugeordnet werden könnte, wäre es geschehen. Es gibt weitere Fehlermeldungen, die nicht zugeordnet werden konnten.

**Comment by mlauss2** [comment]
Habt ihr die Temperaturen des WR zum Zeitpunkt der Meldung notiert?  Bei strahlendem Sonnenschein hab ich diesen Fehler und andere dazu wenn die WR Temperatur >50°C übersteigt.

**Comment by mattespiratos** [comment]
Hi,

bei mir war es ca. 35 Grad und nur einmal kurz früh morgens. Aber erst seit
dem 6 Module dran hängen statt 4.

VG Matthias

Manuel Lauss ***@***.***> schrieb am Mi., 13. Sept. 2023,
13:02:

**Reply by mlauss2** [reply]
Was mir noch einfällt dass das evtl auch eine Überstrom-Warnung des MPPT sein könnte. Meine Panele  könnten mehr Strom (A) liefern als der MPPT verträgt (lt. Datenblättern).
Wie schon beim Threadersteller genannt, funktioniert die Anlage ja auch weiter; ich hatte bisher nur 1x den Fall dass der MPPT in eine Art "Notbetrieb" mit minimaler Leistung ging, was nach einem Neustart auch erledigt war.

**Reply by Alf-Ele** [reply]
Diese Vermutung hatte ich auch schon,  denn ich konnte Ströme nah an der 11,5A Grenze sehen. 
Werden die Fehlercodes eigentlich auch per MQTT in OpenDTU übertragen?

**Reply by huste511** [reply]
Ist den den ein Fehler? OpenDTU benennt aktuell ja alles als "Fehler", selbst den Inverterstart.
Meint es evtl. einfach "max. Leistung erreicht"?
Mal mit Temperaturen oder der Netzspannung in Bezug genommen?
Mit Überstrom wäre ich mir nicht so sicher, da der max. Kurzschlussstrom deutlich höher angegeben ist und div. Leute auch die Hoymiles an Akkus ohne Probleme betreiben.

Gibt es irgendwo eine gute Übersicht bzgl. Hard- und Softwareversionen bzw. die Unterschiede?
